### PR TITLE
Templates

### PR DESCRIPTION
**Pull request by Ikerepc**
Basic about templates and their list with short descripton of each

### CODE REVIEW

File: 1.8/administration/user-titles.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+##### Number of Stars
+
+Here you should put number of stars wich will be shown under user title. If you don't want stars, you will need to put 0 to that filed.
+
+##### Star image
+
+Here you should put star image url. That image will be shown like star in profiles.
+
+## Edit Custom User Title for Specific User
```
Comment (Sama34) [inline 1.8/administration/user-titles.md:47]: Seems empty.

**Comment by Sama34** [comment]
I can't see the explanation of custom template groups. Don't you plan on explaining that?

**Comment by Ikerepc** [comment]
Hi. I created 2 pages here. One for Usertitles and one for Templates. I haven't time to finish it, so I tought to have one docabout ttemplates for themes, and later one for variables, creating templates, custom templates etc.

File: 1.8/administration/user-titles.md (hunk 1 above)
Comment (Ikerepc) [inline 1.8/administration/user-titles.md:47]: Oh, that. I added it, to be easier to navigate, and after that 3 ways to do that. That 3 ways are this sublevels of that... Subtitles. Fell free to edit if you think that it should be different.

**Comment by Ikerepc** [comment]
So is this ok?

**Comment by Sama34** [comment]
Thank you.

**Pull request merged**